src/ot_orchestration/dags/genetics_etl.py

```diff
@@ -10,108 +10,37 @@
-        "destination_bucket": RELEASE_BUCKET_NAME,
-        "destination_object": f"releases/{RELEASE_VERSION}/credible_set/gwas_catalog_curated_studies_susie/",
-    },
     # GWAS Catalog manifest files:
     "gwas_catalog_manifests": {
-        "source_bucket": GWAS_CATALOG_BUCKET_NAME,
-        "source_object": "manifests",
-        "destination_bucket": RELEASE_BUCKET_NAME,
-        "destination_object": f"releases/{RELEASE_VERSION}/manifests/",
-    },
-    # eQTL Catalog study index:
-    "eqtl_catalogue_study_index": {
-        "source_bucket": EQTL_BUCKET_NAME,
-        "source_object": "study_index",
-        "destination_bucket": RELEASE_BUCKET_NAME,
-        "destination_object": f"releases/{RELEASE_VERSION}/study_index/eqtl_catalogue/",
-    },
-    # eQTL Catalog SuSiE credible sets:
-    "eqtl_catalogue_susie_credible_set": {
-        "source_bucket": EQTL_BUCKET_NAME,
-        "source_object": "credible_set_datasets/susie",
-        "destination_bucket": RELEASE_BUCKET_NAME,
-        "destination_object": f"releases/{RELEASE_VERSION}/credible_set/eqtl_catalogue_susie/",
-    },
-    # Finngen study index:
-    "finngen_study_index": {
-        "source_bucket": FINNGEN_BUCKET_NAME,
-        "source_object": f"{FINNGEN_RELEASE}/study_index",
-        "destination_bucket": RELEASE_BUCKET_NAME,
-        "destination_object": f"releases/{RELEASE_VERSION}/study_index/finngen/",
-    },
-    # Finngen SuSiE credible sets:
-    "finngen_susie_credible_set": {
-        "source_bucket": FINNGEN_BUCKET_NAME,
-        "source_object": f"{FINNGEN_RELEASE}/finemapping",
-        "destination_bucket": RELEASE_BUCKET_NAME,
-        "destination_object": f"releases/{RELEASE_VERSION}/credible_set/finngen_susie/",
-    },
-    # UKB PPP eur study index:
-    "UKB_PPP_study_index": {
-        "source_bucket": RELEASE_BUCKET_NAME,
-        "source_object": "output/python_etl/parquet/XX.XX/study_index/ukbiobank",
-        "destination_bucket": RELEASE_BUCKET_NAME,
-        "destination_object": f"releases/{RELEASE_VERSION}/study_index/ukb_ppp_eur/",
-    },
-    # UKB PPP eur credible sets:
-    "ukb_ppp_eur_credible_set": {
-        "source_bucket": RELEASE_BUCKET_NAME,
-        "source_object": "input/credible_set/ukb_ppp_susie",
-        "destination_bucket": RELEASE_BUCKET_NAME,
-        "destination_object": f"releases/{RELEASE_VERSION}/credible_set/ukb_ppp_eur_susie/",
+        "source_bucket": gwas_catalog_manifests_path.bucket,
+        "source_object": gwas_catalog_manifests_path.path,
+        "destination_bucket": release_dir.bucket,
+        "destination_object": f"{release_dir.path}/manifests/",
     },
     # L2G gold standard:
-    "gold_standard": {
-        "source_bucket": RELEASE_BUCKET_NAME,
-        "source_object": "input/l2g/gold_standard/curation.json",
-        "destination_bucket": RELEASE_BUCKET_NAME,
-        "destination_object": f"releases/{RELEASE_VERSION}/locus_to_gene_gold_standard.json",
+    "l2g_gold_standard": {
```

Comment:
Not the purpose of this PR, but this should be moved to the curation repo. It's a static file.

Reply:
I am linking the other PR -> https://github.com/opentargets/curation/pull/36